src/apps/index.tsx

```diff
@@ -36,9 +38,13 @@ const AppDetails: React.FC<RouteComponentProps<{ id: string }>> = ({
   );
 };
 
-const AppDetailsSettings: React.FC<RouteComponentProps<{ id: string }>> = ({
+const AppSettings: React.FC<RouteComponentProps<{ id: string }>> = ({
   match
-}) => <AppDetailsSettingsView id={decodeURIComponent(match.params.id)} />;
+}) => <AppSettingsView id={decodeURIComponent(match.params.id)} />;
+
+const App: React.FC<RouteComponentProps<{ id: string }>> = ({ match }) => (
+  <AppView id={decodeURIComponent(match.params.id)} />
```

Comment:
There should be changes inside AppView to only display "Edit settings" button if configurationUrl is provided.

Reply:
Do you mean AppDetailsView? I've changed it in AppDetailsView.